scripts/components/App.js

```diff
@@ -60,8 +60,15 @@ export class Store extends EventEmitter {
       .then(res => {
         this.state.items.push(res.data);
         this.emit('change', this.state);
-      })
-      .catch(console.error.bind(console));
+      });
+  }
+
+  sync() {
+    return this.collection.sync()
+      .then(res => {
+        this.state.items = this.state.items.concat(res.created);
+        this.emit('change', this.state);
```

Comment:
Nit: I think we went with double quotes for strings. If you prefer simple ones, that's fine; just ensure all double ones are updated.